modules/zookeeper/pom.xml

```diff
@@ -41,48 +41,11 @@
             <version>${project.version}</version>
         </dependency>
 
-        <dependency>
-            <groupId>org.apache.curator</groupId>
-            <artifactId>curator-framework</artifactId>
-            <version>${curator.version}</version>
-        </dependency>
-
-        <dependency>
-            <groupId>org.apache.curator</groupId>
-            <artifactId>curator-recipes</artifactId>
-            <version>${curator.version}</version>
-        </dependency>
-
         <dependency>
```

Comment:
We don't need it in release

We need only `curator-test` in testing dependency

Reply:
@ivandasch this used with scope tests and will not in release